scripts/check-diff-line-width.sh

```diff
@@ -27,10 +27,10 @@ git --no-pager diff $BASE_HASH -U0 -- cdc pkg |
 {
     # Minus 1 for +
     width = length($0) - 1;
-    if (width > 100) {
+    if (width > 120) {
         print "\033[0;31m[ERROR]\033[0m width too long, " length ": " $0 ;
         fail=1 ;
-    } else if (width > 80) {
+    } else if (width > 110) {
```

Comment:
Why change it?

Reply:
it is so small so that if it is a long SQL must be added the ’\n‘. it will be very ugly. 

BTW, the default value in the ide is 120.

Reply:
> it is so small so that if it is a long SQL must be added the ’\n‘. it will be very ugly.

Maybe you not format SQL properly?

Anyway, we may reconsider relaxing go test file limit, for now, let's stick to the rules.

Reply:
I'm sorry I didn't notice it. 😢

Reply:
Maybe we can try https://github.com/segmentio/golines for automation.

Reply:
I will try it when I have time